### ISSUE TITLE
Will you support the official API of ChatGPT?

### ISSUE THREAD

**Issue by MRYingLEE** [jflam/chat-gpt-jupyter-extension#5]
OpenAI just released an official API. Maybe it can make your extension easy to config.

Sometimes, although I have logged in to ChatGPT, your extension couldn't detect my connection. I understand it is not because of your code.
The official API is much faster than the web version.
For privacy, I or my employer don't like to transfer code to a 3rd party beyond OpenAI.

**Comment by nithinreddyyyyyy** [comment]
No, API will cost you more. Rather, you can use ChatGPT Plus which is fast and it is 20$ per month, where as normal API is very costly.
